builtin/providers/github/resource_github_issue_label_test.go

```diff
@@ -134,3 +141,16 @@ resource "github_issue_label" "test" {
   color      = "FFFFFF"
 }
 `, testRepo)
+
+var testAccGitHubIssueLabelExistsConfig string = fmt.Sprintf(`
+// Create a repository which has the default labels
+resource "github_repository" "test" {
+  name = "tf-acc-repo-label-abc1234"
```

Comment:
I think we'll need to randomize this, but apparently the other tests here aren't randomized yet either, so I'll do it in a separate PR.